db/data/regions/TW.yml

```diff
@@ -27,6 +27,12 @@ combined_address_format:
   address2:
     - key: line2
     - key: neighborhood
+      decorator: ","
+  zh-TW:
+    address2:
+      - key: line2
+      - key: neighborhood
+        delimiter: "\u2060"
```

Comment:
This nested override looks fine in YAML but when parsing this its really awkward, at least in Typescript.